data/data/gcp/bootstrap/main.tf

```diff
@@ -0,0 +1,38 @@
+resource "google_compute_instance" "bootstrap" {
+  name         = "${var.cluster_id}-bootstrap"
+  machine_type = var.machine_type
+  zone         = var.zone
+
+  boot_disk {
+    initialize_params {
+      type  = var.root_volume_type
+      size  = var.root_volume_size
+      image = var.image_name
+    }
+  }
+
+  network_interface {
+    subnetwork = var.subnet
+  }
+
+  metadata = {
+    user-data = var.ignition
+  }
+
+  tags = ["${var.cluster_id}-control"]
+
+  labels = var.labels
+}
+
+resource "google_compute_instance_group" "bootstrap" {
```

Comment:
are instance groups a requirement?

Reply:
@abhinavdahiya I'm not sure, we can punt this to CORS-1120